build_library/toolchain_util.sh

```diff
@@ -201,10 +201,10 @@ _configure_sysroot() {
     $sudo eselect profile set --force "$profile"
 
     $sudo tee "${ROOT}/etc/portage/make.conf" >/dev/null <<EOF
-$(portageq envvar -v CHOST CBUILD ROOT SYSROOT \
-    PORTDIR PORTDIR_OVERLAY DISTDIR PKGDIR)
-HOSTCC=\${CBUILD}-gcc
-PKG_CONFIG_PATH="\${SYSROOT}/usr/lib/pkgconfig/"
+$(portageq envvar -v CHOST ROOT SYSROOT PORTDIR PORTDIR_OVERLAY DISTDIR PKGDIR)
```

Comment:
Since portageq will inherit CBUILD from the environment this should work as it was, but the \$ escapes need to remain to make that work so the vars in HOSTCC and such are interpreted later by portage.

Is that to magical?